cns/ipampool/metrics.go

```diff
@@ -6,80 +6,81 @@ import (
 )
 
 const (
-	subnetLabel     = "subnet"
-	subnetCIDRLabel = "subnet_cidr"
+	subnetLabel      = "subnet"
+	subnetCIDRLabel  = "subnet_cidr"
+	podnetARMIDLabel = "podnet_arm_id"
 )
 
 var (
 	ipamAllocatedIPCount = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "ipam_pod_allocated_ips",
+			Name: "cx_ipam_pod_allocated_ips",
```

Comment:
for the cx metrics, can you also add a ConstantLabel in the Opts with key `customer_metric`